# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: ae8e94c8f819
# Full Hash: ae8e94c8f819bd46c31b6fbee1f2cdb5580c9e1f
# Author: Jamie Nicol <jnicol@mozilla.com>
# Date: 2024-07-23 09:34:47
# Regressor Bug: 1898238
# File Overlap Count: 1
# Description:
#   Bug 1898238 - Use SharedSurface_EGLImage for out-of-process webgl and accelerated canvas on Android. r=sotaro,jgilbert
#   
#   Adds a pref wegb.enable-egl-image which is true by default. When
#   enabled, we use EGL images as the preferred texture type for
#   out-of-process webgl and accelerated canvas on Android. In-process
# ==============================================================================

diff -r 549abd685d11 -r ae8e94c8f819 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Mon Jul 22 16:02:34 2024 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Mon Jul 22 16:02:34 2024 +0000
@@ -16298,6 +16298,11 @@
   value: true
   mirror: always
 
+- name: webgl.enable-egl-image
+  type: RelaxedAtomicBool
+  value: true
+  mirror: always
+
 - name: webgl.enable-surface-texture
   type: RelaxedAtomicBool
   value: true
